lib/src/node/render_options.dart

```diff
@@ -13,7 +13,7 @@ class RenderOptions {
   external String get data;
   external dynamic get importer;
   external dynamic get functions;
-  external List get includePaths; // contains Strings
+  external List<Object /* String */ > get includePaths;
```

Comment:
Hmm, why can't it be typed `List<String>`?

Reply:
Because this comes from JS, its runtime is `JSArray`, which doesn't have a reified generic type. Without something like https://github.com/dart-lang/sdk/issues/30554, we have no way of causing it to produce a properly reified type.